### PR TITLE
Dicom ecg extension

### PR DESCRIPTION
**Pull request by ArturRod**
I have created an extension to be able to open and display electrocardiogram (ECG) files, this supports
Sop12LeadECGWaveformStorage: '1.2.840.10008.5.1.4.1.1.9.1.1', --> YES
GeneralECGWaveformStorage: '1.2.840.10008.5.1.4.1.1.9.1.2', --> YES
AmbulatoryECGWaveformStorage: '1.2.840.10008.5.1.4.1.1.9.1.3', --> NO SUPPORT
HemodynamicWaveformStorage: '1.2.840.10008.5.1.4.1.1.9.2.1', --> YES

In my profile, I will update and improve the application, https://github.com/ArturRod/dicom-ecg Once I have the improvements, I will add it to the OHIF viewer.

It looks like this:
image

### CODE REVIEW

**Comment by sedghi** [comment]
Hi, this is awesome contribution!

Couple of notes. As you might have noticed the development on the `master` branch (OHIF v2) has been slowed down and our focus has been shifted to our next version (OHIF v3) which lives on `v3-stable`. The immediate plan is to push v3 to master as soon as possible. You can take a look at the timelines [here](https://ohif.org/roadmap) 

Historically, we have been hesitant to merge stuff into master since then it becomes our responsibility to maintain the code. However, OHIF-v3 has been rearchitectured to allow custom extensions and modes to live on their own respositories and then anyone can install them with OHIF [cli tool](https://v3-docs.ohif.org/development/ohif-cli/#cli-installation). 

In addition, we have added a section in the website that showcase different workflow modes that the community and us has created so far (https://ohif.org/modes), and you can also fill in the application to be added to the list there. 

In summary: if you make this awesome ecg extension/viewport compatible to OHIF-v3 and host it on your own Github, we will index it in the modes section of the website and people can install them with just one command. 

Let me know what you think

**Comment by ArturRod** [comment]
Hi @sedghi 
Thank you very much! So prepare the extension for ohif-v3.
When I have it ready I will upload it as a request! I see that there is also another ECG, even so I will publish it in case you want to add it.
All the best.

**Comment by sedghi** [comment]
I will close this then thanks
